app/templates/components/settings/email-preferences-section.hbs

```diff
@@ -1 +1,66 @@
-<p>Email preferences</p>
\ No newline at end of file
+<div class="ui grid">
+  <div class="ui row">
+    <div class="ui column eight wide">
+      <h4 class="ui header">
+        {{t 'Email Updates are on'}}
+      </h4>
+    </div>
+    <div class="ui column eight wide right aligned">
+      {{ui-checkbox class='toggle checked'}}
+    </div>
+  </div>
+  <div class="ui divider"></div>
+  <div class="ui row">
+    <div class="ui column sixteen wide">
+      <h3 class="ui header">
+        {{t 'Updates about events'}}
+      </h3>
+    </div>
+  </div>
+</div>
+{{#each preferences as |event|}}
+  {{#ui-accordion class='ui segment'}}
+    <div class="ui grid title">
+      <div class="ui column eight wide">
+        <h4 class="ui header">{{event.name}}</h4>
+      </div>
+      <div class="ui column eight wide right aligned">
+        <a class="ui circular label {{if event.status 'green' 'yellow'}}">
+          {{#if event.status}}
```

Comment:
This status is `on` if either one of `isNewPaper`, `isChangeSchedule`, `isNewEvent` is on. If all are off, then this should be off.